Domaca02/src/Domaca02.jl

```diff
@@ -0,0 +1,151 @@
+module Domaca02
+
+"""
+    por = normalPorGost(x)
+
+Vrne vrednost gostote porazdelitve.
+"""
+function normalPorGost(x)
+    return exp(-x^2 / 2) / sqrt(2 * pi)
+end
+
+"""
+    simpson = simpsonovoPravilo(f, a, b, n)
+
+Uporabimo Simpsonovo pravilo za numerično integracijo. Uporabimo vhodno funkcijo `f`
+s krajišči `a` in `b`. Interval med `a` in `b` je razdeljen na `n` delov, katerih
+število mora biti sodo.
+"""
+function simpsonovoPravilo(f, a, b, n)
+    if n % 2 != 0
+        error("n mora biti sodo število!")
+    end
+
+    h = (b - a) / n
+    s = f(a) + f(b)
+
+    for i in 1:2:(n-1)
+        s += 4 * f(a + i * h)
+    end
+    for i in 2:2:(n-2)
+        s += 2 * f(a + i * h)
+    end
+    simpson = s * h / 3
+    return simpson
+end
+
+
+"""
+    normalPor(x; tol=1e-10)
+
+Izračuna vrednost porazdelitvene funkcije standardne normalne porazdelitve F(x).
+Uporablja različne pristope na različnih območjih:
+- za negativne vrednosti uporabi simetrijo F(x) = 1 - F(-x),
+- za srednje vrednosti [-10, 10] uporabi Simpsonovo pravilo,
+- za zelo velike pozitivne vrednosti izračuna repni integral Q(x).
+Relativna napaka < 1e-10 na celem ℝ.
+"""
+function normalPor(x; tol=1e-10)
+    # simetrija za negativne vrednosti
+    if x < 0
+        return 1.0 - normalPor(-x; tol=tol)
+    end
+
+    # rep za zelo velike pozitivne vrednosti
+    if x > 10
```

Comment:
Tule bi bilo smiselno računati vrednosti repa za $x<-10$ in nato vrednosti za $x>10$ izraziti kot $1-F(-x)$.